hack/boilerplate.go.txt

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
This is missing the copyright

Reply:
the boilerplate file is missing the copyright?

Reply:
Yes, see: https://github.com/kubernetes-sigs/cluster-api/blob/master/hack/boilerplate.go.txt

Reply:
This is new from k/k and adapted https://github.com/kubernetes-sigs/cluster-api/tree/master/hack/boilerplate

Reply:
ah right, got it